unstructured/partition/auto.py

```diff
@@ -126,7 +126,7 @@ def partition(
     encoding: Optional[str] = None,
     paragraph_grouper: Optional[Callable[[str], str]] = None,
     headers: Dict[str, str] = {},
-    skip_infer_table_types: List[str] = ["pdf", "jpg", "png"],
+    skip_infer_table_types: List[str] = ["pdf", "jpg", "png", "xls", "xlsx"],
```

Comment:
Could you add in the README that Excel tables are now also not extracted by default?

Reply:
I don't see an existing reference to this in the README. Do you mean in the docs? https://github.com/Unstructured-IO/unstructured/blob/main/docs/source/api.rst#table-extraction-for-other-filetypes

Reply:
ah yes good find! that'll have to be a separate PR for the api readme 👍🏻